dependencies/recommended.txt

```diff
@@ -1,12 +1,13 @@
 # Recommended because some non-commonly-used modules/examples depend on those packages.
 
 -f https://download.pytorch.org/whl/torch_stable.html
-tensorflow
-keras
-torch == 1.6.0+cpu ; sys_platform != "darwin"
-torch == 1.6.0 ; sys_platform == "darwin"
-torchvision == 0.7.0+cpu ; sys_platform != "darwin"
-torchvision == 0.7.0 ; sys_platform == "darwin"
+tensorflow == 2.5.0
+keras == 2.4.3
+tensorboard == 2.5.0
+torch == 1.9.0+cpu ; sys_platform != "darwin"
+torch == 1.9.0 ; sys_platform == "darwin"
+torchvision == 0.10.0+cpu ; sys_platform != "darwin"
+torchvision == 0.10.0 ; sys_platform == "darwin"
```

Comment:
do we need update the dockerfile?

Reply:
I suggest we keep the Dockerfile unchanged for a while.